docs/evm_compatibility/overview.md

```diff
@@ -2,27 +2,51 @@
 title: EVM Compatibility Overview
 ---
 
-Neon EVM is designed to be compatible with the [Ethereum Virtual Machine (EVM)](https://ethereum.org/en/developers/docs/evm/) to provide a as seamless of a developer experience as possible, with some notable differences and considerations. 
+TL;DR
 
-## Shared Features
-Coming from Ethereum, your Solidity smart contracts, development and deployment tools, and standard practices can be moved over to Neon EVM with almost no code changes. Neon EVM accounts follow [Ethereum's account standards](https://ethereum.org/en/developers/docs/accounts/). All [Ethereum opcodes](https://www.evm.codes/?fork=merge) are represented verbatim on Neon EVM. 
+- Apply (most of) Ethereum's standard JSON RPC API methods
+- Direct the calls to Neon's Proxy
+- There are some differences and limitations to consider
 
-### JSON RPC API
-Neon EVM follows Ethereum's standard RPC API and a list of all the JSON RPC API methods that Neon EVM supports can be found [here](./json_rpc_api_methods). They are based on the Ethereum Client API, the Web3 Module API, and the Net Module API.
+## Introduction
```

Comment:
@e-b-e-d-a It is just this page for now

Reply:
Howdy Kseniya -- so this is the only page I have touched. I improved the English/structure but need to know if there are errors/omissions.